reference/entry.json

```
@@ -670,6 +670,108 @@
+        }
+      ],
+      "post": {
+        "summary": "Add a video to an event",
+        "operationId": "post-event-video",
+        "responses": {
+          "200": {
+            "description": "The video was added to the event.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "videoId": {
+                      "type": "string",
+                      "format": "uuid",
+                      "example": "b504cf44-9ab8-4641-9934-38d1cc67242c"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/body/missing\n* https://api.publiq.be/probs/body/invalid-syntax\n* https://api.publiq.be/probs/body/invalid-data",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                },
+                "examples": {
+                  "Invalid data": {
+                    "value": {
+                      "type": "https://api.publiq.be/probs/body/invalid-data",
+                      "title": "Invalid body data",
+                      "status": 400,
+                      "schemaErrors": [
+                        {
+                          "jsonPointer": "/url",
+                          "error": "The required properties url is missing."
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/EventNotFound"
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "type": "string",
+                    "format": "uri",
+                    "example": "https://www.youtube.com/?v=re34sf",
+                    "description": "The URL of the video to add. Currently only Youtube and Vimeo sources are allowed."
+                  },
+                  "copyright": {
+                    "type": "string",
+                    "minLength": 3,
+                    "example": "publiq",
+                    "description": "The copyright holder of the video source."
+                  }
+                },
+                "required": [
+                  "url"
+                ]
+              }
+            }
+          }
+        },
+        "description": "Add a video with a URL to an event",
```

```suggestion
        "description": "Add a video with a URL to an event.",
```
(Same for the place endpoint)

Preferably also add some context here if possible. For example when/why should you include the `copyrightHolder` property? And anything else that is important to know? For example only YouTube and Vimeo URLs are supported. That's also documented in the `url` property but Stoplight doesn't render that very clearly and it's important information to see at a glance imho.

---

I added some extra context. This will require some extra tweaks more likely once a decision is taken about the language property.